app/(afterLogin)/(compensation)/allowance/[id]/_components/allowanceEntitlementSidebar.tsx

```
@@ -142,7 +143,7 @@ const AllowanceEntitlementSideBar = () => {
                 options={allUsers?.items?.map((item: any) => ({
                   ...item,
                   value: item?.id,
-                  label: item?.firstName + ' ' + item?.lastName,
+                  label: item?.firstName + ' ' + item?.middleName + ' ' + item?.lastName,
```

ESLint

[error] Replace `·item?.firstName·+·'·'·+·item?.middleName·+·'·'·+` with `⏎····················item?.firstName·+⏎····················'·'·+⏎····················item?.middleName·+⏎····················'·'·+⏎···················`

(prettier/prettier)